.github/CODEOWNERS

```diff
@@ -3,4 +3,4 @@
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
 # default owners, overridden by directory specific owners below
-* @celo-org/platform-economics
+* @mento-protocol/core
```

Comment:
seems like gh can't find our team. let's check exactly why and get this merged.

Reply:
@ninocomputer it looks like this repo is private -- is that on purpose?